### DISCUSSION TITLE
Cuda 12.3

### DISCUSSION THREAD

**Discussion by Dvalin21** [exadel-inc/CompreFace#1215] [Q&A] [answered]
Just wondering when would Cuda 12.3 be added for use in CompreFace? Right now its 11.8, but my Nvidia uses 12.3.

**Comment by pospielov** [comment, marked as answer]
First, CUDA 11.8 supports a bigger variety of GPUs than 12.3. 
This is a very good reason not to move to Cuda 12.3 for now. 
It will make sense to move when Nvidia produces a new generation of GPUs.
Still, it won't be an easy task. Our code heavily depends on the MXnet deep learning library, and it was officially retired this September. 
And this is another reason not to hurry too much - we must be careful when changing the base framework.
